src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Neoscroll",
+  "description": "A smooth scrolling neovim plugin written in lua",
+  "link": "https://github.com/karb94/neoscroll.nvim",
+  "tags": [
+    "scrolling",
+    "lua"
+  ],
+  "examples": []
```

Comment:
you can put a video / image in there, I saw in your readme you have a vid, put in there if you want